CONTRIBUTING.rst

```diff
@@ -36,3 +36,15 @@ Look in ``.github/workflows/`` to find the commands.
 For example, to run tests::
 
     $ python3 -m pytest
+
+Possible future changes
```

Comment:
I feel like a bullet-point list with more specifics would just duplicate the issue tracker though. I want to avoid adding something that inevitably gets out of date.

Reply:
Okay, then let's just create issues for these two items, and use this section to point people towards the issue tracker.

Reply:
Well, I think these two don't really merit issues. The first is a declaration of policy, and we already have issues for the various specific cases. The second is more of a future risk that doesn't necessarily produce an actionable issue.

Reply:
If the second doesn't translate to an actionable issue, then I don't really think it belongs in README _or_ CONTRIBUTING -- maybe we should just delete it?